components/domains/registerV3.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useContext } from "react";
+import { FunctionComponent, useEffect, useState } from "react";
+import { FormContext } from "@/context/FormProvider";
+import UserInfoForm from "./steps/userInfoForm";
+import { FormType } from "@/utils/constants";
+import CheckoutCard from "./steps/checkoutCard";
+import { useAccount } from "@starknet-react/core";
+import SelectPfp from "./steps/selectPfp";
+import RegisterSteps from "./steps/registerSteps";
+import SearchBar from "../UI/searchBar";
+
+type RegisterV3Props = {
+  domain: string;
+  groups: string[];
+  setDomain: (domain: string) => void;
+};
+
+const discount: Upsell = {
+  duration: 3,
+  paidDuration: 2,
+  maxDuration: 1,
+  discountId: "0",
+  imageUrl: "/register/registerUpsell.webp",
+  title: {
+    desc: "Unlock Extended Domain",
+    catch: "3 Years for the Price of 2!",
+  },
+  desc: "Don't miss out on this one-time offer! This is your chance to secure extended benefits and ensure a lasting digital presence.",
+};
```

Comment:
You can put that in `utils/discounts`

Reply:
You actually have something in `/utils` so I don't understand why it's useful :/

Reply:
I moved it